WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/progress/BackupDownloadProgressFragment.kt

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.jetpack.backup.progress
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import javax.inject.Inject
+
+class BackupDownloadProgressFragment : Fragment(R.layout.backup_download_progress_fragment) {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+    private lateinit var viewModel: BackupDownloadProgressViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val nonNullActivity = requireActivity()
+        (nonNullActivity.application as WordPress).component().inject(this)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initViewModel()
+    }
+
+    private fun initViewModel() {
+        viewModel = ViewModelProvider(
+                this,
+                viewModelFactory
+        ).get(BackupDownloadProgressViewModel::class.java)
```

Comment:
Minor (🔍 ): Consider breaking it down into two lines and by the `.get(...)` instead, like it is used in the builder pattern:
```
        viewModel = ViewModelProvider(this, viewModelFactory)
                .get(BackupDownloadProgressViewModel::class.java)
```

Reply:
It has already been changed in a future branch